database/sql/V050__drop_execution_reward.sql

```diff
@@ -0,0 +1,4 @@
+-- No need to save the CIP14 reward in the database, since it is substituted with CIP20 scheme.
+
+ALTER TABLE order_execution
+DROP COLUMN reward;
```

Comment:
This might come up to validate historical payouts. Let's not delete it yet and ask on slack (`#backend` maybe) what we should do with this data.

Reply:
Reverted dropping the column after discussion on slack.